### PR TITLE
Disable NKRO on V-USB controllers

### PR DESCRIPTION
**Pull request by drashna**
## Description

NKRO isn't available on on V-USB.  Instead of erroring out and failing to compile, just forcibly disable NKRO on V-USB controllers. 

## Types of Changes

- [x] Core
- [x] Enhancement/optimization

## Checklist

- [x] My code follows the code style of this project.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
I wouldn't say NKRO is not supported... it just hasn't been implemented yet. Mainly due to the endpoint mess, but given we can theoretically merge them together with `SHARED_EP_ENABLE` it shouldn't be too much of an issue.

File: tmk_core/common.mk [hunk 1]
```diff
@@ -92,8 +92,12 @@ ifeq ($(strip $(COMMAND_ENABLE)), yes)
 endif
 
 ifeq ($(strip $(NKRO_ENABLE)), yes)
-    TMK_COMMON_DEFS += -DNKRO_ENABLE
-    SHARED_EP_ENABLE = yes
+    ifneq ($(PROTOCOL),VUSB)
+        TMK_COMMON_DEFS += -DNKRO_ENABLE
+        SHARED_EP_ENABLE = yes
+    else
+        $(info NKRO is not supported on V-USB, and has been disabled.)
```
Comment (fauxpark) [inline tmk_core/common.mk:99]:
```suggestion
        $(info NKRO is not currently supported on V-USB, and has been disabled.)
```

**Comment by tzarc** [comment]
So.... what's the direction that we're planning to take? We're okay with disabling, with a note?

**Comment by drashna** [comment]
> So.... what's the direction that we're planning to take? We're okay with disabling, with a note?

I think hard disabling it for now is the best option, until somebody wants to put in the work to add support for it (and more endpoints). 

As for this PR vs #9498, I think this is a much cleaner implementation, as it's a much lighter touch (it doesn't moving things around anywhere else).  However, I am biased, since this is my PR.

**Pull request merged**